bootstrap/templatetags/bootstrap.py

```diff
@@ -90,10 +90,15 @@ def bootstrap_field(field, classes='', template=None, **kwargs):
     extra_classes = getattr(field.field, 'css_classes', [])
     if extra_classes:
         classes += ' ' + ' '.join(extra_classes)
-    # Need to shoehorn some ARIA attributes onto the widget based on information on the field.
+    # Need to shoehorn some ARIA attributes onto the widget based on information on the field
+    labelledby = set(field.field.widget.attrs.get('aria-labelledby', '').split())
+    if widget_class == 'radioselect':
+        labelledby.add(field.auto_id)
+    if labelledby:
+        field.field.widget.attrs['aria-labelledby'] = ' '.join(labelledby)
     describedby = set(field.field.widget.attrs.get('aria-describedby', '').split())
     if field.help_text:
-        describedby.add('%s-help' % field.auto_id)
+        describedby.add(field.auto_id)
```

Comment:
It's my understanding that aria-describedby and aria-labelledby should refer to the IDs of elements that describe  or label the element. Wouldn't this just be saying that the field widget describes itself?